README.md

```diff
@@ -93,4 +93,67 @@ You could use the block-form usage of the form-field component with components t
 
 ## Using your own components
 
-You could use your own components by implementing it in the form-field component like above in the block-form usage. 
+You could use your own components by implementing it in the form-field component like above in the block-form usage.
+
+# Paper-form-field
+Paper-form-field is a variant of the default form-field, using ember-paper for Google Material Design like form-fields. The following sections descripe how the paper-form-field work and what options you can pass to it.
```

Comment:
Split to multiple lines to make it more readable in code (mentioned the max-line length for every other file?)